tests/test_modeling_utils.py

```diff
@@ -787,6 +791,26 @@ def test_legacy_load_from_url(self):
             "https://huggingface.co/hf-internal-testing/tiny-random-bert/resolve/main/pytorch_model.bin", config=config
         )
 
+    def test_save_offloaded_model(self):
```

Comment:
We're almost there! Last little bit here is to use the `@require_torch_non_multi_gpu` decorator on the test so it'll only be ran if a GPU is available.

(Import is `from transformers.testing_utils import require_torch_non_multi_gpu`)

Reply:
Thanks! Added the decorator and import